packages/api-proxy/README.md

```diff
@@ -0,0 +1,156 @@
+# api-proxy
+
+Used for authentication and communication with a [contractsgenerator](https://github.com/leancodepl/contractsgenerator)
+based api.
+
+## Usage
+
+### register
+
+```ts
+import { VerifyOptions } from "jsonwebtoken";
+
+static register({ isGlobal = false, jwtStrategyConfig }: ApiProxySyncConfiguration): DynamicModule
+
+export type ApiProxySyncConfiguration = {
+    isGlobal?: boolean;
+    jwtStrategyConfig: JwtStrategyConfig;
+};
+
+export type JwtStrategyConfig = {
+    jwksUri: string;
+    jsonWebTokenOptions?: VerifyOptions;
+};
+```
+
+##### isGlobal
+
+You can specify if you want to register a module globally.
+
+##### jwksUri
+
+Uri of your auth jwks.
+
+##### jsonWebTokenOptions
+
+`VerifyOptions` is a type imported from `jsonwebtoken` package.
```

Comment:
Here you can either show the `VerifyOptions` type or link to the originical documentation if there's any.